File: Sistem Bilangan/Desimal/Desimal-ke-Biner.cpp
#include <iostream>

int main ()
{
    int desimal = 0;
    std::string biner = "";
    
    std::cout << "Desimal   : ";
    std::cin >> desimal;

    while (desimal > 0)
    {
        int sisa = desimal % 2;
        biner += std::to_string (sisa);
        desimal /= 2;
    }

    std::cout << "Biner     : ";
    for (int i = biner.length () - 1; i >= 0; i--)
        std::cout << biner [i];
    
    return 0;
}

File: Sistem Bilangan/Desimal/DesimalkeBiner.java
import java.util.Scanner;

public class DesimalkeBiner
{
    public static void main (String [] args)
    {
        Scanner scan = new Scanner (System.in);

        System.out.print ("Desimal   : ");
        int desimal = scan.nextInt ();

        StringBuilder biner = new StringBuilder ();
        while (desimal > 0)
        {
            int sisa = desimal % 2;
            biner.insert (0, sisa);
            desimal /= 2;
        }

        System.out.print ("Biner     : " + biner);
    }
}